apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon-page-template.json

```diff
@@ -0,0 +1,8 @@
+{
+	"post_title": "Coming soon",
+	"post_slug": "coming-soon",
+	"category": "coming-soon",
+	"post_content": "<!-- wp:cover {\"customOverlayColor\":\"#117ac9\",\"minHeight\":858,\"contentPosition\":\"center left\",\"align\":\"full\",\"className\":\"is-style-default\"} -->\n<div class=\"wp-block-cover alignfull has-background-dim has-custom-content-position is-position-center-left is-style-default\" style=\"background-color:#117ac9;min-height:858px\"><div class=\"wp-block-cover__inner-container\"><!-- wp:heading {\"align\":\"left\",\"level\":1,\"style\":{\"typography\":{\"fontSize\":60,\"lineHeight\":\"1\"}}} -->\n<h1 class=\"has-text-align-left\" style=\"font-size:60px;line-height:1\">Coming Soon</h1>\n<!-- /wp:heading -->\n\n<!-- wp:paragraph {\"align\":\"left\"} -->\n<p class=\"has-text-align-left\">This site is coming soon</p>\n<!-- /wp:paragraph --></div></div>\n<!-- /wp:cover -->",
+	"hs_template_preview": "https://headstartdata.files.wordpress.com/2020/09/coming-soon.png?w=332",
+	"hs_template_description": "A simple page to let visitors know that your site is coming soon"
+}
```

Comment:
Down the track, would this be better housed in the m1.json (on wpcom), that is, where the other template data lives?

Just asking for later :)

Reply:
hmmm I think so yes, for now it's great here because it can be controleld by the feature flag